texastribune/files
# File Name in Rename Prompts

#### What's this PR do?
- When renaming files/directories, the prompt will contain the original name.
- Adds a makefile that simplifies development!
  - `make start` to...start.
  - `make build` to...build.

#### Why are we doing this? How does it help us?
- When renaming something, a blank prompt is not useful 90% of the time.
- makefiles are awesome.

#### How should this be manually tested?
- make start; your browser should open.
- right-click a folder (should also work for files) and click 'rename'; notice the folders name is there by default.

#### How should this change be communicated to end users?
- Let photo team know!

#### Are there any smells or added technical debt to note?
- Nah

#### What are the relevant tickets?
- https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwvu5Kdv9ASOAfh2/recUnzLx4lTREG1LI?blocks=hide

#### TODOs / next steps:

* [ ] `make build`
